contracts/SwapRBTC.sol

```diff
@@ -128,6 +127,25 @@ contract SwapRBTC is Initializable, OwnableUpgradeable, ISwapRBTC, IERC777Recipi
     return amount;
   }
 
+  function swapRBTCtoSideTokenBtc(uint256 amount, address sideTokenBtcContract) external payable override returns (uint256) {
+    require(enumerableSideTokenBtc.contains(sideTokenBtcContract), "SwapRBTC: Side Token not found");
+    ISideToken rbtc = ISideToken(sideTokenBtcContract);
```

Comment:
side token contract is not rbtc, just the side token that the user should be increased the balance. From what I understand, in this method we receive rBTC and pay that same amount to the user in Side Token. @pmprete correct me if I'm wrong here.